cluster/get-kube.sh

```diff
@@ -143,10 +144,8 @@ fi
 kernel=$(uname -s)
```

Comment:
you could also just execute the TODO and remove these checks entirely

Reply:
rough summary from offline discussion:
- this check causes us to avoid unnecessary download, extract, and run of the next script if the platform is not supported
- this check might also be out of date wrt supported platforms, but:
a) this script should probably go away at some point (when CI no longer depends on it? :grimacing:)
b) probably we should keep this PR scoped to bash lints only

Reply:
I also thought that this looked weird (a bunch of case statements that are empty) but am in favor of lint cleanup and not rocking the boat too much.

Reply:
yeah, in general this _script_ is gross because it's basically a roundabout curl | bash k8s install, but unfortunately CI depends on it and I want to be careful to not break it.

I would be inclined to file a follow-up to refactor this check further, but I think we should focus on not depending on this script. In the meantime though, getting it out of `hack/.shellcheck_failures` will help ensure it doesn't regress in other PRs.

Reply:
cc @krzyzacy we should probably get a "re-implement kubetest extracting kubernetes tarballs to not depend on get-kube.sh" objective planned soon.